src/main/java/core/db/DBUpdater.java

```diff
@@ -641,6 +647,37 @@ private void updateDBv25(int DBVersion, int version) throws SQLException {
 		}
 	}
 
+	private void updateDBv26(int DBVersion, int version) throws SQLException {
```

Comment:
@yaute74 latest stable was using `DBVersion = 24`
Hence, `DBVersion = 25` and corresponding `updateDBv25()` are supposed to ensure transition from HO 1.436 to HO 2.0
Can't you merge your code within `updateDBv25()` instead of introducing a new DB version ?   
@brokenelevator what do you think ?

Reply:
I check with my database in 1.436.
The dbversion is 24 and the column MATCHREPORT, HEIMHATSTATS and GASTHATSTATS in MatchDetails exist.
At the end of update we do a `dbZugriff.saveUserParameter("DBVersion", DBVersion - 1);`  so  updateDBv25 set dbversion to 24..
So it's why I create a new update.

Reply:
the `dbZugriff.saveUserParameter("DBVersion", DBVersion - 1);` is only applied if `HO.isDevelopment() == True`.  It allows for devs to test DBUbdaters during the course of development

Reply:
so basically if you are on test, everytime you launch HO.
it goes through the update (in order to simulate) but does not bump version

Reply:
> the column MATCHREPORT, HEIMHATSTATS and GASTHATSTATS in MatchDetails exist.

correct, this is because @brokenelevator fixed a mistake that was made when I created updateDBv24

> MATCHREPORT was made longer here but not in table definition to fix this mistake and not repeat code updateDBv24 is falling through to updateDBv25

Reply:
ok I misread. I will correct it